mods/d2k/chrome/ingame-observer.yaml

```diff
@@ -0,0 +1,994 @@
+									Align: Right
+									Shadow: True
+								Label@SPENT:
+									X: 475
+									Y: 0
+									Width: 80
+									Height: PARENT_BOTTOM
+									Align: Right
+									Shadow: True
+								Label@HARVESTERS:
+									X: 555
+									Y: 0
+									Width: 80
+									Height: PARENT_BOTTOM
+									Align: Right
+									Shadow: True
+						ScrollItem@PRODUCTION_PLAYER_TEMPLATE:
+							X: 0
+							Y: 0
+							Width: 400
+							Height: 25
+							BaseName: scrollitem-nohover
+							Children:
+								ColorBlock@PLAYER_COLOR:
+									X: 0
+									Y: 0
+									Width: PARENT_RIGHT - 200
+									Height: PARENT_BOTTOM
+								GradientColorBlock@PLAYER_GRADIENT:
+									X: PARENT_RIGHT - 200
+									Y: 0
+									Width: 200
+									Height: PARENT_BOTTOM
+								Image@FLAG:
+									X: 2
+									Y: 2
+									ImageName: random
+									ImageCollection: flags
+								Label@PLAYER:
+									X: 35
+									Y: 0
+									Width: 120
+									Height: PARENT_BOTTOM
+									Font: Bold
+									Shadow: True
+								ObserverProductionIcons@PRODUCTION_ICONS:
+									X: 155
+									Y: 0
+									Width: 0
+									Height: PARENT_BOTTOM
+									TooltipContainer: TOOLTIP_CONTAINER
+						ScrollItem@SUPPORT_POWERS_PLAYER_TEMPLATE:
+							X: 0
+							Y: 0
+							Width: 400
+							Height: 25
+							BaseName: scrollitem-nohover
+							Children:
+								ColorBlock@PLAYER_COLOR:
+									X: 0
+									Y: 0
+									Width: PARENT_RIGHT - 200
+									Height: PARENT_BOTTOM
+								GradientColorBlock@PLAYER_GRADIENT:
+									X: PARENT_RIGHT - 200
+									Y: 0
+									Width: 200
+									Height: PARENT_BOTTOM
+								Image@FLAG:
+									X: 2
+									Y: 2
+									ImageName: random
+									ImageCollection: flags
+								Label@PLAYER:
+									X: 35
+									Y: 0
+									Width: 120
+									Height: PARENT_BOTTOM
+									Font: Bold
+									Shadow: True
+								ObserverSupportPowerIcons@SUPPORT_POWER_ICONS:
+									X: 155
+									Y: 0
+									Width: 0
+									Height: PARENT_BOTTOM
+									TooltipContainer: TOOLTIP_CONTAINER
+						ScrollItem@ARMY_PLAYER_TEMPLATE:
+							X: 0
+							Y: 0
+							Width: 400
+							Height: 25
+							BaseName: scrollitem-nohover
+							Children:
+								ColorBlock@PLAYER_COLOR:
+									X: 0
+									Y: 0
+									Width: PARENT_RIGHT - 200
+									Height: PARENT_BOTTOM
+								GradientColorBlock@PLAYER_GRADIENT:
+									X: PARENT_RIGHT - 200
+									Y: 0
+									Width: 200
+									Height: PARENT_BOTTOM 
+								Image@FLAG:
+									X: 2
+									Y: 2
+									Width: 35
+									Height: PARENT_BOTTOM - 4
+									ImageName: random
+									ImageCollection: flags
+								Label@PLAYER:
+									X: 35
+									Y: 0
+									Width: 120
+									Height: PARENT_BOTTOM
+									Font: Bold
+									Shadow: True
+								ObserverArmyIcons@ARMY_ICONS:
+									X: 155
+									Y: 1
+									Width: 0
```

Comment:
This is missing the `IconWidth: 30` that was the most important part!

Reply:
That will remove the black border at the bottom.

Reply:
Current layout (`IconWidth: 32`):
<img alt="Screenshot 2020-02-09 at 12 37 06" src="https://user-images.githubusercontent.com/167819/74102186-06f28e80-4b39-11ea-8786-453092368495.png">

Notice how the count overlay runs off the right edge of the icon, which happens because the d2k icons are smaller than the 64px width used in TD/RA.

With `IconWidth: 30`:

<img alt="Screenshot 2020-02-09 at 12 37 46" src="https://user-images.githubusercontent.com/167819/74102191-11ad2380-4b39-11ea-8e72-4b177991c563.png">